ext/mbstring/mbstring.c

```diff
@@ -1473,6 +1473,27 @@ static PHP_INI_MH(OnUpdate_mbstring_http_output_conv_mimetypes)
 	return SUCCESS;
 }
 /* }}} */
+
+#if HAVE_MBREGEX
+/* {{{ static PHP_INI_MH(OnUpdate_regex_stack_limit */
+static PHP_INI_MH(OnUpdate_regex_stack_limit)
+{
+	zend_long stack_limit;
+
+	stack_limit = atol(ZSTR_VAL(new_value));
+	if (stack_limit > 0 && stack_limit <= UINT_MAX) {
+		onig_set_match_stack_limit_size(stack_limit);
```

Comment:
We should probably use onig_set_match_stack_limit_size_of_match_param() and onig_match_with_param()